src/librustdoc/html/static/storage.js

```diff
@@ -35,17 +35,20 @@ var localStoredTheme = getSettingValue("theme");
 
 var savedHref = [];
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
There's no eslint directive to say that these functions are "exported"?

Reply:
Apparently not... :-/